schema/objects/transactions/issuance/StockIssuance.schema.json

```diff
@@ -40,15 +40,7 @@
     },
     "cost_basis": {
```

Comment:
I'm thinking cost_basis should be nullable. I think we all agreed on votes per share being required, but not cost basis.
If we follow the discussion link https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/discussions/218 the concern was around needing the field at all, and was agreed that having the field was necessary for handling odd cases where cost_basis is not/no longer calculable as quantity * share_price.

But I wouldn't say that's enough to make it a required field. If it's null we could assume the cost basis is quantity * share_price

Reply:
That works for me. So you're thinking we'll have it be a required field with null or a value instead of an optional field?

Reply:
It should be a regular optional field, which means removing "cost_basis" from the required array down below

Reply:
Cool @jacobyavis. Easy fix. Give me a sec.

Reply:
Fix is in, @jacobyavis